packages/executor/src/modules/eth.ts

```diff
@@ -194,12 +194,12 @@ export class Eth {
 
     // calculate callGasLimit based on paid fee
     const { cglMarkup } = this.config;
-    const totalGas: BigNumber = BigNumber.from(paid).div(userOp.maxFeePerGas);
-    let callGasLimit = totalGas
-      .sub(preOpGas)
-      .mul(10000 + this.config.cglMarkupPercent)
-      .div(10000) // % markup
-      .add(cglMarkup || 0);
+    // const totalGas: BigNumber = BigNumber.from(paid).div(userOp.maxFeePerGas);
```

Comment:
I would use this as the default behaviour on chains that do not have an implementation address. So instead of commenting this, please handle it as a fallback option.